src/styles/card.scss

```diff
@@ -0,0 +1,86 @@
+.card {
+  width: var(--card-width);
+  border: 1px solid var(--border-color);
+  border-radius: 5px;
+  padding: 32px 16px 16px;
+  margin: var(--margin);
+  transition-duration: 300ms;
+
+  &:hover {
+    cursor: pointer;
+    transform: scale(120%);
```

Comment:
The scale function in the transform property should have a value less than or equal to 1 to avoid oversizing. A value of 1.2 means 120% which is an increase. Consider changing it to something like 'transform: scale(1.2);' for a subtle growth on hover.